lib/service/service.go

```diff
@@ -394,12 +396,12 @@ func (process *TeleportProcess) initAuthService() error {
 			srv.Addr = fmt.Sprintf("%v:%v", process.Config.AdvertiseIP.String(), port)
 		}
 		for !askedToExit {
+			sleepTime := defaults.ServerHeartbeatTTL/2 + utils.RandomDuration(defaults.ServerHeartbeatTTL/10)
+			time.Sleep(sleepTime)
```

Comment:
what's the purpose of this change? looks like now you start with sleep, while before it was the opposite, just not sure what does it affect really